debian/templates

```diff
@@ -1,11 +1,11 @@
-Template: matrix-synapse/server-name
+Template: matrix-synapse-py3/server-name
```

Comment:
it feels like a mistake to rename the config settings. The existing config settings for the existing package can equally well apply to the new package, and it will save the user having to answer the question again when switching.